RELEASE_NOTES.md

```diff
@@ -9,7 +9,7 @@
 - The minimum supported version of `protobuf` was bumped to 5.28.1 and to 1.66.1 for `grpcio`. Make sure to update your dependencies accordingly.
 - The `AddComponentBoundsRequest.validity_duration` field was replaced by `AddComponentBoundsRequest.request_lifetime` to match the fields in `SetComponentPowerActiveRequest` and `SetComponentPowerReactiveRequest`.
 - The `AddComponentBoundsResponse.ts` field was renamed to `AddComponentBoundsResponse.valid_until` for extra clarity and also to match the fields in `SetComponentPowerActiveResponse` and `SetComponentPowerReactiveResponse`.
-- The version of `frequenz-api-common` has been updated to v0.7.0. Please refer to the [release notes](https://github.com/frequenz-floss/frequenz-api-common/releases/tag/v0.7.0) and update your code accordingly.
+- The version of `frequenz-api-common` has been updated to v0.8.0. Please refer to the [release notes](https://github.com/frequenz-floss/frequenz-api-common/releases/tag/v0.8.0) and update your code accordingly.
```

Comment:
Maybe it would be useful to provide the original version, because users will actually need to go through v0.7 changes too, as they are not included in v0.8 release notes, and probably they will need to go through any other intermediate version between the one used in the current microgrid API version and the upcoming one.

Reply:
Good point. Will submit the changes in a bit.

Reply:
OK, make sure to re-request a review so I get notified.